lang/ftidy/Portfile

```
@@ -25,63 +26,29 @@ long_description \
     of keywords or non-keywords, and do various other clean-up \
     tasks on FORTRAN-77 source programs.
 
-checksums \
-    md5 f940dfa793408368f92af6aab535619b \
-    sha1 200e4531f172355cde49c455e2f7e9df8f1f8404
+checksums md5    a2611f7448119f9f0982ddfd3e68140c \
+          sha1   82da71cac7380016233ab2aa217dbf3a0dd3686f \
+          rmd160 1b96ed6a8e0ad787bdd46c21fbded3f445df7336 \
+          sha256 b08d174bdaf16baa2d6b614115f3b568587c9e635392899a8d6cc9cf1828a98c \
+          size   133309
 
-patchfiles              tidy.for.patch
+patchfiles       tidy.for.patch
 
-if {![variant_isset gcc43] && ![variant_isset gcc44] && ![variant_isset gcc45] && ![variant_isset gcc46] && ![variant_isset gcc47] && ![variant_isset gcc48] && ![variant_isset gcc49] && ![variant_isset gcc5] && ![variant_isset gcc6] && ![variant_isset gcc7]} {
-    default_variants    +gcc6
-}
-
-variant gcc43 conflicts gcc44 gcc45 gcc46 gcc47 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc43} {
-    configure.compiler  macports-gcc-4.3
-}
-
-variant gcc44 conflicts gcc43 gcc45 gcc46 gcc47 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc44} {
-    configure.compiler  macports-gcc-4.4
-}
-
-variant gcc45 conflicts gcc43 gcc44 gcc46 gcc47 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc45} {
-    configure.compiler  macports-gcc-4.5
-}
-
-variant gcc46 conflicts gcc43 gcc44 gcc45 gcc47 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc46} {
-    configure.compiler  macports-gcc-4.6
-}
-
-variant gcc47 conflicts gcc43 gcc44 gcc45 gcc46 gcc48 gcc49 gcc5 gcc6 gcc7 description {Build using gcc47} {
-    configure.compiler  macports-gcc-4.7
-}
-
-variant gcc48 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc49 gcc5 gcc6 gcc7 description {Build using gcc48} {
-    configure.compiler  macports-gcc-4.8
-}
-
-variant gcc49 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc48 gcc5 gcc6 gcc7 description {Build using gcc49} {
-    configure.compiler  macports-gcc-4.9
-}
-
-variant gcc5 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc48 gcc49 gcc6 gcc7 description {Build using gcc5} {
-    configure.compiler  macports-gcc-5
-}
-
-variant gcc6 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc48 gcc49 gcc5 gcc7 description {Build using gcc6} {
-    configure.compiler  macports-gcc-6
-}
-
-variant gcc7 conflicts gcc43 gcc44 gcc45 gcc46 gcc47 gcc48 gcc49 gcc5 gcc6 description {Build using gcc7} {
-    configure.compiler  macports-gcc-7
-}
+compilers.choose f77
+compilers.setup  require_fortran
+use_configure    no
 
-build.pre_args        ${configure.fflags} ${configure.f77_archflags}
+build.pre_args   ${configure.fflags} ${configure.f77_archflags}
```

If you would like to make whitespace changes here to align these like the rest of the file, that would be fine. But much more than that and I would ask you to make a separate whitespace-only commit so that it is easier to see the functional changes in their own commit.
 ```suggestion
build.pre_args          ${configure.fflags} ${configure.f77_archflags}
```

---

OK